### DISCUSSION TITLE
Recordings active objects

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5656] [Ask A Question]
Does the "active_object" recording mode records all active/moving objects that qualify as events, or does it only capture any active/moving object in the frame? I was trying to set a pre and post capture for the recordings, similar to the settings available for events, but couldn't find any such options. This made me wonder if the "active_object" Recording mode would be based on events that have pre and post capture settings.

**Comment by NickM-27** [comment]
I think https://docs.frigate.video/configuration/record#what-do-the-different-retain-modes-mean should answer your question otherwise I'm not sure what you're asking. 

The active_objects mode means any 10 second segment that contains an object that moved will be kept

**Reply by jsbowling42** [reply]
so the scenario is i have required zones setup for my event retention so i only get events in the UI for objects that enter my zones. but what im trying to do is make sure the recording captures the video before and after that event for some duration. i just didn't know how to best setup the record retention; or if there was a setting similarly to the pre and post capture for the event would accomplish this?

**Reply by NickM-27** [reply]
The 24/7 recording does not care about required_zones, that only applies to events 

The pre and post is what would do it for the events to ensure that time is recorded as part of the event

**Reply by jsbowling42** [reply]
Yeah I was just trying to save space lol and set the recording to something other than “all”.  I was hopping there was a way to only save the recordings of objects with some sort of pre and post capture of that object. As a stop gap for not having recorded resolution switching for events.
